modules/agent-install-about-mirroring-for-disconnected-registry.adoc

```diff
@@ -35,6 +35,8 @@ mirrors:
 virthost.ostest.test.metalkube.org:5000/localimages/local-release-image
 source: registry.ci.openshift.org/ocp/release
 ----
+For multi-arch release, append `--keep-manifest-list` option to oc command.
```

Comment:
@aleskandro @zniu1011 could you please check if multi-arch should be mentioned here?

Reply:
it would also be worth adding a note with a bit of detail on what it does, i.e something like
"--keep-manifest-list" would mirror the release payload manifestlist containing payloads for 4 architectures (amd64, arm64, ppc64le, s390x) in case of multi-arch payloads. Note that this would mean an increase in payload size by 4x of a single arch payload

Reply:
This is an agent-related doc and we don't support it yet.